Ensure primary key field is auto-injected when absent from the fields array (#25077)
directus/directus: .changeset/famous-turtles-bow.md

---
'@directus/api': patch
---

> Fixed primary key field was not auto injected when not present in fields array

Reviewer: ```suggestion
Ensured primary key field is auto-injected when absent from the fields array
```
